tests/test_spherePoint.py

```diff
@@ -481,18 +502,6 @@ def testInquality(self):
                 self.assertEqual((point1 != point2), (point2 != point1))
                 self.assertNotEqual((point1 == point2), (point1 != point2))
 
-    def testBearingToError(self):
-        """Test if bearingTo() correctly handles invalid input.
-        """
-        northPole = SpherePoint(0.0, 90.0, degrees)
-        southPole = SpherePoint(0.0, -90.0, degrees)
-        safe = SpherePoint(0.0, 0.0, degrees)
-
-        with self.assertRaises(pexEx.DomainError):
-            northPole.bearingTo(safe)
-        with self.assertRaises(pexEx.DomainError):
-            southPole.bearingTo(safe)
-
     def testBearingToValue(self):
         """Test if bearingTo() returns the expected value.
         """
```

Comment:
I would like to see some test coverage at the poles. Does `bearingTo` produce the output it should (especially as a function of the pole's longitude)?

Reply:
Fair enough. I had not noticed until now that the existing test are all on the equator.